packages/core/src/MobileCore.ts

```diff
@@ -36,6 +37,7 @@ interface IMobileCore {
   setAppGroup: (appGroup?: string) => void;
   resetIdentities: () => void;
   clearUpdatedConfiguration: () => void;
+  initialize: (initOptions: InitOptions, callback: (error: any, result: any) => void) => void;
```

Comment:
Make it consistent with other functions by returning a Promise<void>.

Reply:
Can you also add initializeWithAppId(appId: String) -> Promise<void>?

See the flutter implementation [here](https://github.com/adobe/aepsdk_flutter/pull/129/files#diff-74260df90d944c608d1db76d0a15513fc22ac75d41a2e66fa244bc158f839422R26)